scripts/lib-franklin.js

```diff
@@ -157,6 +157,34 @@ export function toCamelCase(name) {
   return toClassName(name).replace(/-([a-z])/g, (g) => g[1].toUpperCase());
 }
 
+/**
+ * Helper function to streamline creation of new HTML elements
+ * @param tag Tag name, like 'div'
+ * @param attributes Attributes associated with the element, like class
+ * @param html Either html or array of elements to be added as children
+ * @returns {Element} HTML element
+ */
+export function createTag(tag, attributes, html) {
```

Comment:
alternatively, we can use DOM Helpers [1] which offer some more sugar syntax like `li(div(),div())`

[1] https://main--franklin-jsx-like--andreituicu.hlx.live/

Reply:
> alternatively, we can use DOM Helpers [1] which offer some more sugar syntax like `li(div(),div())`
> 
> [1] https://main--franklin-jsx-like--andreituicu.hlx.live/
> 
> edit: they're also part of https://www.hlx.live/developer/block-collection#block-party

Tried it - didn't felt that it increased readability (at least on my changes).